tests/components/sensibo/conftest.py

```diff
@@ -43,3 +46,19 @@ async def load_int(hass: HomeAssistant) -> MockConfigEntry:
     await hass.async_block_till_done()
 
     return config_entry
+
+
+@pytest.fixture(name="get_data")
+async def get_data_from_library(
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
+) -> SensiboData:
+    """Retrieve data from upstream Sensibo library."""
+
+    data_fixture = load_fixture("data.json", "sensibo")
```

Comment:
I'd put the fixture loading and json parsing in a separate pytest fixture that isn't run in the test event loop.

Reply:
I did now put it separate. Is this how you meant?